docker/metrictank/metrictank.ini

```diff
@@ -93,7 +83,7 @@ proftrigger-heap-thresh = 10000000
 log-min-dur = 5min
 
 # only log log-level and higher. 0=TRACE|1=DEBUG|2=INFO|3=WARN|4=ERROR|5=CRITICAL|6=FATAL
-log-level = 2
+log-level = 1
```

Comment:
why this change? i find debug to be overwhelming 99% of the time i run this stack.